### PR TITLE
Add models: ElasticNet, Ridge, Lasso and LinearSVC

### PR DESCRIPTION
**Pull request by Juan-M-V**
Implement the fuzzers for the linear models:
- lasso
- ridge
- elsatic_net
- SVC

### CODE REVIEW

**Review by daphneherlambda — CHANGES_REQUESTED** [review]
Just to keep the code uniform, add the comments in the linear_svc to all the fuzzers

**Comment by Juan-M-V** [comment]
> Just to keep the code uniform, add the comments in the linear_svc to all the fuzzers

Resolved: https://github.com/lambdaclass/concrete-ml-fuzzer/pull/11/commits/6dfffc1b1847746840ff4193b93dc0a69c733fcc

**Review by daphneherlambda — APPROVED** [review]

File: src/elastic_net.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+concrete_model, scikit_model, data_info = initialize_models(ElasticNet)
+
+def compare_models(input_bytes):
+    # Run the inference, encryption and decryption is done in the background
+    data = consume_bytes(input_bytes, data_info, margin=0.1)
+    # Get scikit prediction
+    FHE_pred = concrete_model.predict(data, execute_in_fhe=True)
+    # Get the mean percentage error to make sure the accuaracy is around 98%
+    prediction = scikit_model.predict(data)
+
```
Comment (jrchatruc) [inline src/elastic_net.py:15]: I think comments here should be one line below each, right?

**Comment by jrchatruc** [comment]
Looks good other than my small comment

**Review by jrchatruc — APPROVED** [review]

**Pull request merged**